gui/kitchen-info.lua

```diff
@@ -221,6 +221,14 @@ local function findStringInLine(x1, x2, y, str)
     return -1, -1
 end
 
+local function pluginCompatibility_search_isInputActive()
+    return false  -- FIXME: Implement when API is available
```

Comment:
Are you planning on implementing this? As it stands, this PR looks good except for the P functionality. I'd suggest removing the P toggle for now so we can merge this PR, then we can add it back if and when the search plugin gains the appropriate API.